avocado/core/nrunner.py

```diff
@@ -643,14 +644,28 @@ class Task:
     While a runnable describes what to be run, and gets run by a
     runner, a task should be a unique entity to track its state,
     that is, whether it is pending, is running or has finished.
-
-    :param identifier:
-    :param runnable:
     """
 
     def __init__(self, identifier, runnable, status_uris=None,
```

Comment:
I think we should set `identifier` as default. It doesn't make a sense to me to set it to `None` when I want to create a Task. We should use this :

`def __init__(self, runnable, identifier=None, status_uris=None, known_runners=None):`

Reply:
I agree with @richtja too. Having the first argument as required when we set a default value, seems strange to me.

But this change would require modifying the API here. So not sure how " public" Task it is. I have a feeling that only Avocado internal code is using it, but I'm not sure.